src/graphql.rs

```diff
@@ -415,9 +409,21 @@ where
 
         let (nodes, has_more) = if let Some(before) = before {
             let to = latest_key(&before)?;
-            iter_to_nodes_with_filter(iter, &to, cmp::Ordering::is_le, filter, first)
+            iter_to_nodes_with_filter(
+                iter,
+                &to,
+                cmp::Ordering::is_le,
+                filter,
+                first.to_usize().unwrap_or_default(),
```

Comment:
I think this can be just `first`, after we have `first` as `usize`.

Reply:
For this one, since `first` is `Option<i32>`, I came up with this solution. What do you think?

<img width="1920" alt="Screenshot 2024-04-06 at 3 12 04 AM" src="https://github.com/aicers/review-web/assets/102562226/7c2ce1c2-1810-4b87-8fe4-1c1ce10afece">

Reply:
Looks good!

Reply:
Perfect will push and rebase now. I thought I changed it and included it on my previous commit but I must've forgot my apologies.

Reply:
Done!